packages/integration_test/android/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.3'
+        classpath 'com.android.tools.build:gradle:4.1.1'
```

Comment:
I am not entirely sure how version resolving for gradle works for this testbed. I will get someone from Flutter core team to review these particular changes. That's likely what's causing your build failure.

Reply:
@cyanglaz Could you please take a look at the build file changes in this PR? What is the proper way to up the gradle version?

Reply:
I revert the changes to the `integration_test` package in this PR. This appears to prevent those failed builds.